brave/brave-core
# Tor connection progress improved. Added slow phase.

Resolves https://github.com/brave/brave-browser/issues/24978

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Toggle Use bridges
2. Select built-in meek-azure
3. Open Tor window
4. Wait 45 seconds, 'Still connecting' message should be shown
5. Wait more 45 seconds, Error message should be shown
